### PR TITLE
🔧 bsx classic mint no royalty

### LINKED ISSUE #4189: Royalty toggle reversed behaviour
### What happened?

Royalty toggle turns on royalty when it's off and vice versa

### Please reproduce in steps

Mint NFT -> use royalty toggle

### Expected Behavior

When I turn toggle ON, i will be able to set royalty

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

_No response_

### Which wallet you are using?

_No response_

### Screenshots

image

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by roiLeo**
continue from #4655 

## PR Type

- [x] Bugfix

## Context

- [x] Closes #4189 
- [x] remove unused code

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

## Screenshot 📸

Screenshot 2023-01-10 at 17-17-53 KodaDot Low fees and low carbon minting KodaDot - Kusama NFT Market Explorer

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]
code lgtm

File: components/bsx/Create/CreateToken.vue [hunk 1]
```diff
@@ -147,23 +152,23 @@ export default class CreateToken extends mixins(
 ) {
   @Prop({ type: Boolean, default: false }) showExplainerText!: boolean
 
-  protected base: BaseTokenType<MintedCollection> = {
+  public base: BaseTokenType<MintedCollection> = {
```
Comment (vikiival) [inline components/bsx/Create/CreateToken.vue:155]: Why change of visibility?

Comment (roiLeo) [reply components/bsx/Create/CreateToken.vue:155]: linter not happy
`Property 'base' is protected and only accessible within class 'CreateToken' and its subclasses`

Comment (vikiival) [reply components/bsx/Create/CreateToken.vue:155]: 🫡 Hvala!

**Comment by vikiival** [comment]
pay 50 usd

**Pull request merged**